client/db/src/lib.rs

```diff
@@ -62,12 +64,14 @@ impl DatabaseSettingsSrc {
 }
 
 pub(crate) mod columns {
-	pub const NUM_COLUMNS: u32 = 4;
+	pub const V1_NUM_COLUMNS: u32 = 4;
+	pub const NUM_COLUMNS: u32 = 5;
 
 	pub const META: u32 = 0;
 	pub const BLOCK_MAPPING: u32 = 1;
 	pub const TRANSACTION_MAPPING: u32 = 2;
 	pub const SYNCED_MAPPING: u32 = 3;
+	pub const ETHEREUM_SCHEMA_CACHE: u32 = 4;
```

Comment:
Can we reuse `META` column instead? Since this is single value it should already be sufficient.

Reply:
You can add the function in `MetaDb`, similar to how it's done for `current_syncing_tips`.